src/components/Authentication/LoginForm.tsx

```diff
@@ -64,9 +66,10 @@ const LoginForm: React.FC = () => {
   const { isLoginIn, triggerLogin } = useLogin();
   const theme = React.useContext(ThemeContext);
   const { lastUsername, setLastUsername } = React.useContext(UserContext);
+  const { isTemporarlyBlocked } = React.useContext(LoginTemporisationContext);
   const keyboard = useKeyboard();
 
-  return (
+  return !isTemporarlyBlocked ? (
```

Comment:
I would maybee handle this with an early return, don't know if it is better, but I guess it specifies that this is not the normal happy path
```jsx
if (isTemporarlyBlocked) {
    return <LoginTemporisation />;
}
```